src/lib.rs

```diff
@@ -29,11 +29,21 @@ use cosmwasm_vm::{
 };
 
 use crate::args::{
-    ARG1, ARG2, CACHE_ARG, CODE_ID_ARG, ENV_ARG, GAS_USED_ARG, INFO_ARG, MSG_ARG, WASM_ARG,
+    ARG1, ARG2, CACHE_ARG, CODE_ID_ARG, CONTRACT_INFO_ARG, ENV_ARG, GAS_USED_ARG, INFO_ARG,
+    MSG_ARG, WASM_ARG,
 };
 use crate::cache::{cache_t, to_cache};
 use crate::error::{handle_c_error, Error};
 
+#[derive(Copy, Clone)]
+#[repr(C)]
+/// A set of flags exposing some aspects of the contract configuration
+pub struct ContractFlags {
+    pub ibc_enabled: bool,
+    pub stargate: bool,
```

Comment:
Why does the chain care about this information (`stargate`)? It already compiles the VM with stargate support, and configures supported features `NewVM` such that requirement and feature is checked by cosmwasm-vm.

Reply:
I was just trying to add something here besides ibc_enabled to demo what is possible.
AFAIK nothing is really required

Reply:
Alright, let's remove it than. Good to have the option to expend this even with one element.